css-highlight-api-1/Overview.bs

```diff
@@ -383,11 +383,11 @@ Priority of Overlapping Highlights</h4>
 	<div class=example id=overlap-highlight-ex>
 		<xmp highlight=html>
 			<style>
-				p::highlight(foo) {
+				::highlight(foo) {
```

Comment:
Can you update this to :root::highlight(foo)?

Reply:
Makes sense, updated.